src/AI/AICombatUtils.java

```diff
@@ -254,7 +254,7 @@ public static int findMaxMobileWeaponRange(Commander co)
         }
         double thisShot =
             CombatEngine.simulateBattleResults(attacker, target, gameMap,
-                                               space.xCoord, space.yCoord).defenderHPLoss;
+                                               space.xCoord, space.yCoord).defender.deltaPreciseHP * -1;
```

Comment:
The intent might be more clear if you use `Math.abs` instead of just multiplying an expected negative result by -1.